auth_user_case_insensitive/tests/test_res_users.py

```diff
@@ -20,13 +18,13 @@ def setUp(self):
         self.model_obj = self.env["res.users"]
 
     def _new_record(self):
-        """ It should enerate a new record to test with """
+        """It should enerate a new record to test with"""
         partner_id = self.env["res.partner"].create(self.partner_vals)
         self.vals["partner_id"] = partner_id.id
         return self.model_obj.create(self.vals)
 
     def test_login_is_lowercased_on_create(self):
-        """ It should verify the login is set to lowercase on create """
+        """It should verify the login is set to lowercase on create"""
```

Comment:
So if changing docstring, add period at the end, to make it pydocstyle257 compatible. Also docstring should start with a verb for method. Instead of `It should verify`, use `Verify the login is set..`